README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Version npm](https://img.shields.io/npm/v/nodify-shopify.svg)](https://www.npmjs.com/package/nodify-shopify)
 [![Build Status](https://img.shields.io/travis/microapps/Nodify-Shopify/master.svg)](https://travis-ci.org/microapps/Nodify-Shopify)
+[![bitHound Score](https://www.bithound.io/github/microapps/Nodify-Shopify/badges/score.svg)](https://www.bithound.io/github/microapps/Nodify-Shopify/)
+[![Dependency Status](https://david-dm.org/microapps/Nodify-Shopify.svg "Dependencies Checked & Updated Regularly (Security is Important!)")](https://david-dm.org/microapps/Nodify-Shopify)
```

Comment:
Hi there @lpinca and @PauloLuan – Carlos here from bitHound. I just wanted to let you know there is currently a [PR pending into shields.io](https://github.com/badges/shields/pull/452) for the bitHound badge. It seems to be stalled a bit, so I recommend using the bitHound hosted version until it is merged. Also, I just wanted to point out we also have dependencies badges in case you wanted to check them out: https://www.bithound.io/github/microapps/Nodify-Shopify/master/dependencies/npm